Thales/thales_safenet/_meta/manifest.yml

```
@@ -0,0 +1,8 @@
+---
+uuid: 00dfc292-410b-4850-9065-8b410db5ce85
+name: Thales Safenet MFA
```

```suggestion
name: Thales SafeNet Trusted Access
```
Is that the right product name ?